qemu/tests/live_snapshot_basic.py

```diff
@@ -51,7 +51,8 @@ def get_snapshot_file(self):
         """
         Get path of snapshot file.
         """
-        snapshot_file = "images/%s" % self.snapshot_file
+        image_format = self.params["image_format"]
+        snapshot_file = "images/%s.%s" % (self.snapshot_file, image_format)
```

Comment:
@yanan-fu well, previous code generate snapshot file path without image_format, so the snapshot file will be `images/sn1` , when I try to reopen guest with snapshot file, pass image name `sn1` to vm.create(), it will generate image file with image_format like `images/sn1.qcow2`. That's why here add image_format as suffix to image file path, so that it fit framework logic and get correct file path when reopen guest with snapshot file.